### DISCUSSION TITLE
Using TinyUSB (not Adafruit)

### DISCUSSION THREAD

**Discussion by sonik-br** [earlephilhower/arduino-pico#2657] [Q&A] [answered]
On arduino ide, it's possible to use TinyUSB on the core without requiring Adafruit_TinyUSB_Arduino?
I would like to have more control over the usb withut doing heavy modifications to the adafruit lib.

**Comment by earlephilhower** [comment, marked as answer]
Select the Tools->USB Stack->None from the menus and you can write whatever you want for USB, but you obviously need to manage it all on your own including the IRQs, callbacks, locking, etc.

**Reply by sonik-br** [reply]
Gonna try. Thanks!
